tests/operators/test_regrid.py

```diff
@@ -152,10 +153,53 @@ def test_regrid_onto_xyspacing_unknown_method(regrid_source_cube):
 
 
 @pytest.mark.filterwarnings("ignore:Selected point is within")
-def test_regrid_to_single_point(cube):
+def test_regrid_to_single_point_east(cube):
     """Regrid to single point."""
     # Test extracting a single point.
-    regrid_cube = regrid.regrid_to_single_point(cube, 0.5, 358.5, "Nearest")
+    # Test that the grid latitude rotation works when the
+    # centre of the grid is too far east (note that the
+    # test cube, by default, is too far east, centred east
+    # of 180 deg).
```

Comment:
It would be good to put this information about what the test is doing inside the docstring (The triple quoted string at the start of the function), rather than in comments.

This makes it show up in more useful places, such as alongside failing tests.

Reply:
The text has been moved into the doc-string and description made to read more clearly.